wee_slack.py

```diff
@@ -1067,16 +1067,16 @@ def command_talk(current_buffer, args):
     server = servers.find(current_domain_name())
     if server:
         channel = server.channels.find(args)
-        if not channel:
+        if channel is not None:
```

Comment:
Can we avoid double negation here? It's much easier to read if you swap the two arms of the if around and drop the `not`.